html/donate/en.html

```diff
@@ -132,22 +137,19 @@ <h3>Help us to fund the Open Food Facts 2022 budget!</h3>
       </div>
 
       <div
-        class="columns large-5 xlarge-6 xxlarge-7 hide-for-medium hide-for-small"
+        class="columns medium-6 large-5 xlarge-6 xxlarge-7 hide-for-small"
       >
         <h2 style="margin-bottom:1rem">
           A message from the Open Food Facts team
         </h2>
 
-<div class="panel callout" style="margin-top:0.5rem">
-<h3>Help us to fund the Open Food Facts 2022 budget!</h3>
-<p>Open Food Facts is 100% free and independent of the food industry. We need your help to continue and to grow the project.</p>
-</div>
-
-        <img
-          src="https://static.openfoodfacts.org/images/svg/crowdsourcing-icon.svg"
-          style="width:240px;float:left;margin-right:5px;"
-          alt="Yes we scan! logo"
-        />
+        <div class="panel callout" style="margin-top:0.5rem">
```

Comment:
```suggestion
        <div class="panel callout" style="margin-top:0.5rem;padding-bottom:0.5em;">
```

Reply:
The padding becomes asymetric, it feelds weird. We reduced the text quite a lot already, let's see how it looks now.